setup.py

```diff
@@ -1,4 +1,7 @@
 from setuptools import setup
 
 
-setup(use_scm_version=True)
+setup(
+    name="conda_lock",
```

Comment:
Note that this is not really needed for the package but for GitHub. It cannot compute the projects that depends on this without the name in setup.py.